### DISCUSSION TITLE
No snapshots in notifications

### DISCUSSION THREAD

**Discussion by TallWilber** [dgreif/ring#657] [Q&A] [answered]
I have the Ring Video Doorbell Wired, and I am getting alerts and doorbell ring notifications through HomeKit. However, the notification don't have a snapshot of the video included in the notification. I have another Ring camera, the Indoor Cam, and that camera does send a snapshot of the video in the notification. The only relevant message in the logs is this:

`[5/25/2021, 8:16:08 AM] [Ring] Front Door Detected Motion`
`[5/25/2021, 8:16:49 AM] [Ring] Snapshot for Front Door (doorbell_graham_cracker - Unknown Model) failed to refresh after 70 attempts.  This is normal behavior since this camera is unable to capture snapshots while streaming`
`[5/25/2021, 10:23:47 AM] [Ring] Garage Detected Motion. Loading snapshot before sending event to HomeKit`

Update: I've gotten numerous notifications without the snapshot, but I just got a notification with a snapshot. However, the Homebridge logs look identical:
`[5/25/2021, 1:08:06 PM] [Ring] Front Door Detected Motion`
`[5/25/2021, 1:08:47 PM] [Ring] Snapshot for Front Door (doorbell_graham_cracker - Unknown Model) failed to refresh after 70 attempts.  This is normal behavior since this camera is unable to capture snapshots while streaming`

**Comment by dgreif** [comment, marked as answer]
As the logs state, `This is normal behavior since this camera is unable to capture snapshots while streaming`.  See https://github.com/dgreif/ring/wiki/Snapshot-Limitations#snapshots-while-recording for more details
